src/app/settings/user/template.html

```diff
@@ -27,10 +27,6 @@
 </div>
 
 <mat-card>
-  <mat-card-header>
```

Comment:
why this is removed?

Reply:
because as @cschieder suggest to remove the "Account" headline and the thin line separator so it looks like this,
![177564923-da5b72cc-e58b-4514-a22f-8158edab81a5](https://user-images.githubusercontent.com/85109141/179221351-d4d91b0e-3b24-4d9c-b576-beeab8242627.png)